python/gps/algorithm/config.py

```diff
@@ -18,13 +15,13 @@
     'init_traj_distr': None,  # A list of initial LinearGaussianPolicy
                               # objects for each condition.
     # Trajectory optimization.
-    'traj_opt': TrajOptLQRPython({}),
```

Comment:
I think this is okay. In practice, we'll always specify this in the hyperparams anyway.